server/channels/api4/cloud.go

```diff
@@ -57,7 +57,7 @@ func (api *API) InitCloud() {
 	// GET /api/v4/cloud/cws-health-check
 	api.BaseRoutes.Cloud.Handle("/check-cws-connection", api.APIHandler(handleCheckCWSConnection)).Methods("GET")
 
-	api.BaseRoutes.Cloud.Handle("/delete-workspace", api.APISessionRequired(selfServeDeleteWorkspace)).Methods(http.MethodDelete)
+	api.BaseRoutes.Cloud.Handle("/delete-workspace", api.APISessionRequired(selfServeDeleteWorkspace)).Methods("DELETE")
```

Comment:
Our govet tool wasn't prepared to handle an expression -- I took the easy route to fix this.

Reply:
Seems like we will need to write another checker to find these :p